core/src/main/resources/error/error-classes.json

```diff
@@ -1883,6 +1883,13 @@
     ],
     "sqlState" : "42K05"
   },
+  "RESOLVED_PLAN_HAVE_CONFLICTING_ATTRS" : {
```

Comment:
Can you trigger this errors from user space? Could you add some tests, please.

Reply:
OK